docs/data/intake-api/generated/rum_v3_events.ndjson

```diff
@@ -1,6 +1,3 @@
+{"me": {"y": {"t": "Processing"},"sa": {"ysc": {"v": 1},"yss": {"v": 124}}}}
```

Comment:
is this a valid metricset? We can send metricset inside transaction or even in the same events as of transaction?

Reply:
Asking because how do you differentiate them from custom metricsets like CPU, Memory etc.

Reply:
https://github.com/elastic/apm-server/pull/3733/#discussion_r419396153

:see_no_evil:

Reply:
I don't think it is needed to differentiate, we just accept both, but if it comes inside the transactions then we fill in the transaction info

Reply:
maybe you (and @simitt ) were thinking about having different, separated specs for system and breakdowns, but I think that would just complicate things (more types internally, etc).

Reply:
Cool, thanks for the details Juan.